scout/commands/view/case.py

```diff
@@ -52,7 +52,7 @@ def cases(institute, display_name, case_id, nr_variants, variants_treshold, simi
 
     else:
         if display_name:
-            name_query = f"case:{display_name}"
+            name_query = ImmutableMultiDict({"case": display_name})
         models = adapter.cases(collaborator=institute, name_query=name_query)
```

Comment:
Ah, this works since the `populate_` function is fixed, but we should also fix the type hint here! 😸 
https://github.com/Clinical-Genomics/scout/blob/2d8210a95ae49f515d003288d898f8588eb534de/scout/adapter/mongo/case.py#L367

Reply:
Right, I'll fix it!